core/server/data/schema/schema.js

```diff
@@ -346,8 +346,7 @@ module.exports = {
         // NOTE: integration_id column needs "nullable: true" -> "nullable: false" migration (recreate table with nullable: false)
         // CASE: Ghost instances initialized pre 4.0 will have this column set to nullable: true in db schema
         integration_id: {type: 'string', maxlength: 24, nullable: false, references: 'integrations.id', cascadeDelete: true},
-        // @TODO: add isIn validation
-        status: {type: 'string', maxlength: 50, nullable: false, defaultTo: 'available'},
+        status: {type: 'string', maxlength: 50, nullable: false, defaultTo: 'available', validations: {isIn: [['available']]}},
```

Comment:
Why do we need the field if we don't use it?

Reply:
Good point - I've switched it to dropping the column